### PR TITLE
Dashboard page

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -243,12 +243,7 @@ const HomePage = ({ services }: any) => {
...
-            <Link
-              href={t(
-                'common:home_page.contribute.buttonbloc2.button.href',
-                'https://github.com/ambanum/OpenTermsArchive'
-              )}
-            >
+            <Link href="https://github.com/ambanum/OpenTermsArchive">
               <a
                 title={t(
                   'common:home_page.contribute.buttonbloc2.link.title',
```
Comment (martinratinaud) [inline src/pages/index.tsx:249]: bloc**k**

File: src/pages/index.tsx [hunk 2]
```diff
@@ -270,12 +265,7 @@ const HomePage = ({ services }: any) => {
             iconName="FiDatabase"
             iconColor="#0496FF"
```
Comment (martinratinaud) [inline src/pages/index.tsx:266]: Can't we have the colors straight from the theme in javascript?

File: public/locales/catchAll/common.json [hunk 3]
```diff
@@ -143,5 +132,29 @@
   "subscribe_form.changes.before": "For this document, you should be interesssed to",
   "subscribe_form.fields.service.optgroup.gatekeepers": "Gatekeepers",
   "subscribe_form.fields.service.optgroup.otherservices": "Other services",
-  "home_page.contributors.subtitle": "Thanks 🙏"
+  "home_page.contributors.subtitle": "Thanks 🙏",
+  "header.link.home.title": "Home",
+  "header.link.adddocument.title": "Add a document",
+  "header.link.home": "Home",
+  "header.link.adddocument": "Add a document",
```
[the remark this answers is not in the document]
Comment (clementbiron) [reply public/locales/catchAll/common.json:139]: I didn't understand !

File: src/modules/Common/components/ServicesList.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+      <Column width={100} title={title} subtitle={subtitle}>
+        <ul className={s.listServices_items}>
+          {Object.keys(services || {}).map((serviceName: string) => {
+            const documentTypes = (services || {})[serviceName];
+            return (
+              <li className={s.listServices_item} key={`service_list_${serviceName}`}>
+                <div className={s.listServices_item_service}>{serviceName} :</div>
+                <div className={s.listServices_item_docs}>
+                  {documentTypes.map((documentType: string) => {
+                    const target = `https://github.com/ambanum/OpenTermsArchive-versions/blob/master/${serviceName}/${documentType}.md`;
```
Comment (martinratinaud) [inline src/modules/Common/components/ServicesList.tsx:34]: In fact Services List does not give me the information on what I'm looking at, which is the latest version recorded also



What would be more interesting would be to have all the latest changes
- for one service: https://github.com/ambanum/OpenTermsArchive-versions/commits/master/Apple
- for one document https://github.com/ambanum/OpenTermsArchive-versions/commits/master/Apple/Privacy%20Policy.md

And maybe all the snapshots or at least the last one

Maybe a table alignment would be thus more appropriated


Like so
Apple - link to "All versions" - link to "All snapshots"
| Document      | Latest version  | Latest snapshot  | All versions  | All snapshots  |
| ----------- | ----------- |----------- |----------- |----------- |
| Privacy Policy      | link       | link       |link       | link       |
| Terms of use      | link       | link       |link       | link       |
| Cookie Policy      | link       | link       |link       | link       |

**Pull request merged**